### PR TITLE
[bitnami/minio] Make runAsNonRoot configurable

### PR DESCRIPTION
**Pull request by ckotzbauer**
Signed-off-by: Christian Kotzbauer christian.kotzbauer@gmail.com

**Description of the change**
Add "runAsNonRoot" config-option to the securityContext.

**Benefits**
It is possible to set "runAsNonRoot" to "true" now.

**Possible drawbacks**

**Applicable issues**

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/minio/README.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ The following table lists the configurable parameters of the MinIO&reg; chart an
 | `securityContext.enabled`         | Enable security context                                                                   | `true`                         |
 | `securityContext.fsGroup`         | Group ID for the container                                                                | `1001`                         |
 | `securityContext.runAsUser`       | User ID for the container                                                                 | `1001`                         |
+| `securityContext.runAsNonRoot`    | Avoid running as root User                                                                | `false`                        |
```
Comment (carrodher) [inline bitnami/minio/README.md:121]: Running containers as non root is a best practices in terms of securizing containers, so I think the default approach should be the opposite, by default the security context should be set to use `true` as value for `runAsNonRoot`
```suggestion
| `securityContext.runAsNonRoot`    | Run container as non-root user                                                        | `true`                         |
```

+info: https://docs.bitnami.com/tutorials/work-with-non-root-containers/

File: bitnami/minio/values.yaml [hunk 2]
```diff
@@ -224,6 +224,7 @@ securityContext:
   enabled: true
   fsGroup: 1001
   runAsUser: 1001
+  runAsNonRoot: false
```
Comment (carrodher) [inline bitnami/minio/values.yaml:227]: See above description
```suggestion
  runAsNonRoot: true
```

**Review by carrodher — CHANGES_REQUESTED** [review]
Thanks for your contribution, please, take a look at my comments

**Comment by ckotzbauer** [comment]
@carrodher I addressed your comments.

**Review by ckittelmann — APPROVED** [review]

**Review by carrodher — APPROVED** [review]
LGTM, thanks. I just launched our internal test & release pipeline, if everything works as expected the PR should be automatically closed after some time

**Comment by ckotzbauer** [comment]
@carrodher  Thanks for your review. Is there any problem in the publish-pipeline? The changes from this PR are still not published.

**Comment by carrodher** [comment]
Yes, it's true. We test all our charts on top of different k8s clusters such as GKE, AKS, TKG, IKS, etc.

In this case, there is an issue with IKS. It is not related to the PR but the cluster infrastructure itself, once solved, the chart should be automatically released. Sorry for the delay

**Comment by ckotzbauer** [comment]
Thanks for your fast reply. That's okay, just a bit intransparent for contributors 😄 😉

**Comment by carrodher** [comment]
You can find more info about this process in the contributing guideline, see https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#pr-approval-and-release-process 😉

**Comment by carrodher** [comment]
The new version should be available after running `helm repo update`:
```console
$ helm search repo bitnami/minio
NAME         	CHART VERSION	APP VERSION	DESCRIPTION
bitnami/minio	6.8.0        	2021.5.27  	Bitnami Object Storage based on MinIO(R) is an ...
```

Thanks again for your contribution!

**Pull request merged**